qt/scientific_interfaces/Inelastic/Corrections/CalculatePaalmanPings.ui

```diff
@@ -44,7 +44,7 @@
 &lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
 p, li { white-space: pre-wrap; }
 &lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
-&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Deprecation warning:&lt;/span&gt; This feature has been deprecated as of version 6.10. If you still use this feature, please inform a Mantid developer. Assuming no user feedback, this feature will be removed in version 6.12 (in two minor releases time).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
+&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Deprecation warning:&lt;/span&gt; This feature has been deprecated as of version 6.10. If you still use this feature, please inform a Mantid developer. Assuming no user feedback, this feature will be removed in version 6.12 (in two minor releases time). Please note that the &amp;quot;Calculate Monte Carlo Absorption&amp;quot; tab offers a far better version of this feature.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
_please inform a Mantid developer._ -> Is it worth adding here support mail?

_Assuming no user feedback, this feature will be removed in version 6.12 (in two minor releases time)*_-> Can this be : 
 _Assuming no user feedback, this feature will be removed two minor versions from now (in version 6.12)_